tests/qa_automation/qa_run.pm

```diff
@@ -0,0 +1,100 @@
+package qa_run;
```

Comment:
IMO this file should be in `lib/` dir. Or we'll need to add tests/qa_automation to `@INC` which I think is ugly.